starter-code/src/clue.js

```diff
@@ -82,13 +123,98 @@ name: Observatory
 name: Theater
 name: Guest House
 name: Patio
-*/
+ */
+
+
+const DiningRoom = {
+    name: "Dining Room"
+};
+const Conservatory = {
+    name: "Conservatory"
+};
+const Kitchen = {
+    name: "Kitchen"
+};
+const Study = {
+    name: "Study"
+};
+const Library = {
+    name: "Library"
+};
+const BilliardRoom = {
+    name: "Billiard Room"
+};
+const Lounge = {
+    name: "Lounge"
+};
+const Ballroom = {
+    name: "Ballroom"
+};
+const Hall = {
+    name: "Hall"
+};
+const Spa = {
+    name: "Spa"
+};
+const LivingRoom = {
+    name: "Living Room"
+};
+const Observatory = {
+    name: "Observatory"
+};
+const Theater = {
+    name: "Theater"
+};
+const GuestHouse = {
+    name: "Guest House"
+};
+const Patio = {
+    name: "Patio"
+};
 
 // Characters Collection
-const charactersArray = [];
+const charactersArray = [mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
 
```

Comment:
you could've also used the push method here: charactersArray.push(mrGreen, drOrchid...)